modules/modules.sqf

```diff
@@ -14,13 +14,15 @@
 #include "start_on_safe\root.sqf"
 #include "start_with_earplugs\root.sqf"
 
-//Additional modules that can be enabled by removing the // 
+//Additional modules that can be enabled by removing the //
 //!!Some of them require changing settings in their settings.sqf file!!
 
 //#include "a3_endscreen\root.sqf"
+//#include "aceMedical\root.sqf"
```

Comment:
It's for the most part duplicate of `medical_settings`. Delete one of them. If you leave `aceMedical`, I recommend you change it's folder name to `ace_medical`. Why? To keep all folder names in same convention (`lower_case`), so it's harder to make a mistake when writing an `#include`. We don't want that to happen because it makes game crash and is pretty annoying.